components/feedstock-card.js

```diff
@@ -89,13 +103,18 @@ export const FeedstockCard = ({ feedstock }) => {
   return (
     <>
       <Box
+        ref={cardRef}
         id={id} // Use the generated id
         sx={{
           mb: [7, 7, 7, 8],
           width: '100%',
           display: 'flex',
           flexDirection: 'column',
           position: 'relative',
+          transition: 'background-color 0.3s ease',
+          backgroundColor: isSelected ? 'highlight' : 'transparent',
```

Comment:
This color is not very readable against some of the feedstock colors. I'd experiment with options that don't introduce such intense contrast non-neutral colors against each other -- maybe try using the feedstock color instead?
```suggestion
          backgroundColor: isSelected ? alpha(color, 0.05) : 'transparent',
```

Other options:
- Show focus using a border instead of background color.
- Use a more neutral color from the theme instead of `highlight` (e.g., `muted` or `hinted`).
- Lower the opacity of the `highlight` waaay down, like to `0.1`.